swagger/api/v1/definition.yaml

```diff
@@ -0,0 +1,23 @@
+get:
+  operationId: getPresentationDefinition
+  summary: Get a Presentation Definition with the ID.
+  parameters:
+    - name: id
+      in: path
+      description: Presentation Definition ID
+      required: true
+      schema:
+        type: string
+  responses:
+    '200':
+      description: Provided Presentation Definition associated with the Presentation Definition ID.
+      content:
+        application/json:
+          schema:
+            $ref: './api/models/presentation_definition_wrapper.yaml#/presentation_definition_wrapper'
+    '400':
```

Comment:
Given this is a REST API with an id in the path, I would also expect an explicit 404 response